src/viam/sdk/common/grpc_client_fwd.hpp.in

```diff
@@ -0,0 +1,55 @@
+#pragma once
+
+#cmakedefine VIAMCPPSDK_GRPCXX_LEGACY_CLIENT_FWD
+
+#ifndef VIAMCPPSDK_GRPCXX_LEGACY_CLIENT_FWD
+
+// Forward declaration file for grpc ClientContext and ClientReaderInterface<T>.
+// This file provides includes for recent (>= 1.32.0) versions of grpc.
+
+namespace grpc {
```

Comment:
(q) why is the namespace `grpc` here but `grpc_impl` in the `else` case below?

Reply:
In recent versions of grpc the classes are defined in `namespace grpc`, but in the oldest two versions we support the class definitions are in `namespace grpc_impl` and `typedef`'d into `namespace grpc`, which is incompatible with forward declaring the classes in `namespace grpc`